### PR TITLE
feat: remove SafeHtml type check to unblock users of ivy renderer

### LINKED ISSUE #709: Toastr not working on 9.0.0-next.4
When using ngx-toastr 10.2.0 on Angular 9.0.0-next.4 I get the error message

`ERROR Error: "Template error: Can't bind to 'ngIf' since it isn't a known property of 'button'."

Angular 3
        createUnknownPropertyError
        elementPropertyInternal
        property
    Toast_Template ngx-toastr.js:978
`

and

`ERROR Error: "Template error: Can't bind to 'ngIf' since it isn't a known property of 'div'."`

I have a pretty standard installation. Does someone have any ideas on how to prevent this error?

The Toastr appears but has no text in it

### PR DESCRIPTION
**Pull request by Coooi**
Updated to Angular 8.2.12;
Added aot=true to ng serve;
Set enableIvy to true on tsconfig.app.json.

### CODE REVIEW

**Comment by Coooi** [comment]
@yarrgh @scttcper @renovate-bot 
Would you please approve this? 
We're using it on our project.

**Comment by Coooi** [comment]
@peterblazejewicz @chris-allen @yutamago @PhilippStein @rkettelerij @exequiel09 
Could any of you take a look please?

**Comment by Coooi** [comment]
> Is that change of `SafeHtml` due to this?
> [angular/angular#33028](https://github.com/angular/angular/issues/33028)
> Thanks!

Exactly, since the html string is being sanitized when `config.enableHtml` is true, so I removed SafeHtml type from there.

File: angular.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "ngx-toastr:build"
+            "browserTarget": "ngx-toastr:build",
+            "aot": true
```
[the remark this answers is not in the document]
Comment (Coooi) [reply angular.json]: @peterblazejewicz thank you for the review.
I've committed the changes.

**Comment by scttcper** [comment]
Can we replace it with any for the time being

**Comment by Coooi** [comment]
@scttcper Hi Scott, since the field only expects a String, I believe it will be fine.
The string is being sanitized anyway when `config.enableHtml` is true.

**Comment by Coooi** [comment]
@scttcper @peterblazejewicz 
Any change we get this deployed today?
Our team depends on ngx-toastr, and we need to make a production build today.
I'd appreciate it, thanks.

**Pull request merged**